provider/virtiofs_utils.py

```diff
@@ -0,0 +1,46 @@
+"""
+Module for providing interfaces about testing virtio-fs.
+
```

Comment:
this file can accommodate other AUX functions like run_fio , mount_ etc....

Reply:
For the mount and run_fio are not only for virtio-fs, they cound appied other scenarios that need them. This module just put interfaces for testing virtio-fs.

Reply:
BTW, if there is `xxx` feature also need run_fio mount_ ect,  do you will new a module to add functions same as the viriofs_utils? actually they do the same things.
Like the run_fio, munt_, they are the common api. so I don't suggest making them proprietary.